ext/pdo_sqlite/tests/pdo_sqlite_filename_uri.phpt

```diff
@@ -5,6 +5,11 @@ pdo_sqlite
 --FILE--
 <?php
 
+// create with in-memory database
+$db = new PDO('sqlite:file::memory:?cache=shared');
+
+var_dump($db->exec('CREATE TABLE test1 (id INT);'));
```

Comment:
Should this also open a second connection to check that it is actually shared?

Reply:
It could. I don't know if php is responsible for all the uri options working though. I was mainly testing the special case filenames.

Reply:
Given that we don't directly pass the DSN to SQLite3, that additional check would make sense. It may also make sense to add an [INI section](http://qa.php.net/phpt_details.php#ini_section) with `open_basedir=`, because if open_basedir is set, `file:` URIs are not supported by PHP.

Reply:
FreeBSD fails for some reason. What is the recommended open_basedir setting and then target directory to use to trigger a fail on all platforms?

Reply:
I think you could use `open_basedir=.` for that.

Reply:
> I think you could use `open_basedir=.` for that.

And write to what directory? If it doesn't restrict and creates a file, does it matter where it goes?

Reply:
Found an example test to follow.